docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -247,6 +247,10 @@ All fields in the ``DOCUMENTATION`` block are lower-case. All fields are require
         - ref: aci_guide
           description: Detailed information on how to manage your ACI infrastructure using Ansible.
 
+        # Reference by rST document name
+        - doc: /user_guide/become
```

Comment:
@acozine Do we want to use this form of referencing to a file name?
This means we can't move files around. I thought we should use `:ref:`

Reply:
```
[dag@moria ansible.git]$ grep -r ':doc:' docs/docsite/ | grep ':doc:' | wc -l
183
```

Reply:
```
[dag@moria ansible.git]$ grep -r ':doc:' docs/docsite/rst/modules/ | grep ':doc:' | wc -l
3
```

Reply:
In this case jumping directly to "Become and Windows" would be even better.

Reply:
Fixed.